test/OpenTelemetry.AutoInstrumentation.Tests/Configuration/PluginsConfigurationHelperTests.cs

```diff
@@ -28,42 +29,51 @@ public class PluginsConfigurationHelperTests
     {
-        var action = () => Sdk.CreateTracerProviderBuilder().InvokePlugins(new[] { "Missing.Assembly.PluginType, Missing.Assembly" });
+        var tracerAction = () => Sdk.CreateTracerProviderBuilder().InvokePlugins(new[] { "Missing.Assembly.PluginType, Missing.Assembly" });
+        tracerAction.Should().Throw<FileNotFoundException>();
 
-        action.Should().Throw<FileNotFoundException>();
+        var meterAction = () => Sdk.CreateMeterProviderBuilder().InvokePlugins(new[] { "Missing.Assembly.PluginType, Missing.Assembly" });
+        meterAction.Should().Throw<FileNotFoundException>();
     }
 
     [Fact]
     public void MissingPluginTypeFromAssembly()
     {
-        var action = () => Sdk.CreateTracerProviderBuilder().InvokePlugins(new[] { "Missing.PluginType" });
+        var tracerAction = () => Sdk.CreateTracerProviderBuilder().InvokePlugins(new[] { "Missing.PluginType" });
+        tracerAction.Should().Throw<TypeLoadException>();
 
-        action.Should().Throw<TypeLoadException>();
+        var meterAction = () => Sdk.CreateMeterProviderBuilder().InvokePlugins(new[] { "Missing.PluginType" });
+        meterAction.Should().Throw<TypeLoadException>();
     }
 
     [Fact]
     public void PluginTypeMissingExpectedMethod()
     {
         var pluginAssemblyQualifiedName = GetType().AssemblyQualifiedName;
-        var action = () => Sdk.CreateTracerProviderBuilder().InvokePlugins(new[] { pluginAssemblyQualifiedName });
+        var tracerAction = () => Sdk.CreateTracerProviderBuilder().InvokePlugins(new[] { pluginAssemblyQualifiedName });
+        tracerAction.Should().Throw<MissingMethodException>();
 
-        action.Should().Throw<MissingMethodException>();
+        var meterAction = () => Sdk.CreateMeterProviderBuilder().InvokePlugins(new[] { pluginAssemblyQualifiedName });
+        meterAction.Should().Throw<MissingMethodException>();
     }
 
     [Fact]
     public void PluginTypeMissingDefaultConstructor()
     {
         var pluginAssemblyQualifiedName = typeof(MockPluginMissingDefaultConstructor).AssemblyQualifiedName;
-        var action = () => Sdk.CreateTracerProviderBuilder().InvokePlugins(new[] { pluginAssemblyQualifiedName });
+        var tracerAction = () => Sdk.CreateTracerProviderBuilder().InvokePlugins(new[] { pluginAssemblyQualifiedName });
+        tracerAction.Should().Throw<MissingMethodException>();
 
-        action.Should().Throw<MissingMethodException>();
+        var meterAction = () => Sdk.CreateMeterProviderBuilder().InvokePlugins(new[] { pluginAssemblyQualifiedName });
+        meterAction.Should().Throw<MissingMethodException>();
     }
 
     [Fact]
     public void InvokePluginSuccess()
     {
         var pluginAssemblyQualifiedName = typeof(MockPlugin).AssemblyQualifiedName;
         Sdk.CreateTracerProviderBuilder().InvokePlugins(new[] { pluginAssemblyQualifiedName });
+        Sdk.CreateMeterProviderBuilder().InvokePlugins(new[] { pluginAssemblyQualifiedName });
```

Comment:
`NotThrow()` could be used here to indicate what this test is checking

reference: https://fluentassertions.com/exceptions/